tests/test_class_attributes.rs

```diff
@@ -113,6 +110,52 @@ fn test_fallible_class_attribute() {
     }
 
     Python::with_gil(|py| {
-        py.get_type::<BrokenClass>();
+        let stderr = CaptureStdErr::new(py).unwrap();
+        let _ = std::panic::catch_unwind(|| py.get_type::<BrokenClass>());
```

Comment:
Couldn't at least assert directly that a panic did happen?

Reply:
Good idea 👍